paper/paper.md

```diff
@@ -0,0 +1,107 @@
+Additionally, we have implemented comprehensive air-sea flux models [e.g. @wanninkhof:1992] and sediment models [e.g. @soetaert:2000] which can easily be applied to tracers in the models.
+We focus on the simulation of idealized sub-mesoscale systems, but this flexible framework allows users to model problems of any scale.
+This framework is made possible by our contributions to ``Oceananigans.jl``, adding a streamlined user interface to swap biogeochemical models with no modification to other model configurations.
+This interface also facilitates rapid prototyping, as models can be implemented and swapped easily by just extending a few key functions.
+This flexibility and ease-of-use is unmatched in existing biogeochemical models.
+
+``OceanBioME.jl`` was designed specifically to study ocean carbon dioxide removal (OCDR) strategies.
+Assessing the effectiveness and impacts of OCDR is challenging due to the complexities of the interactions between the biological, chemical, and physical processes involved in the carbon cycle.
+Moreover, field trials of OCDR interventions are generally small-scale and targeted, while the intervention required to have a climate-scale impact is regional or global.
+We have built ``OceanBioME.jl`` to meet these challenges by creating tools that provide a modular interface to the different components within the ocean modelling framework provided by ``Oceananigans.jl``.
+This allows easy access to a suite of biogeochemical models ranging from simple idealized to full-complexity models.
+\autoref{fig2} shows a simple column model with an OCDR intervention (macroalgae growth) added after a warm-up period, which increases the carbon export of the system.
+
+![Here we show the results of a 1D model, forced by idealised light and mixing, which qualitatively reproduces the biogeochemical cycles in the North Atlantic.
+We then add kelp (500 frond / m² in the top 50 m of water) in December of the 2ⁿᵈ year (black vertical line) which causes an increase in air-sea carbon dioxide exchange and sinking export, as well as a change in the phytoplankton growth cycle.
+Figure made with `Makie.jl` [@makie]. \label{fig2}](column_example.png)
+
+The biologically active particles built into ``OceanBioME.jl`` are particularly useful for OCDR applications.
+Accurate carbon accounting is essential for assessing the effectiveness of OCDR strategies.
+Biologically active particles can be used to track carbon from a particular source while accounting for interactions with its surroundings.
+Biologically active particles can also be used to model OCDR deployment strategies including seaweed cultivation, alkalinity enhancement, and marine biomass regeneration.
+``OceanBioME.jl`` currently includes an extended version of the sugar kelp model presented by @broch:2012 as an example of the utility and implementation of these features.
```

Comment:
This section is kind of a statement of need, plus maybe the final paragraph. but I'm not really sure how to separate it fully into it's own section